### PR TITLE
extracting venue to be used by the client side

### PR DESCRIPTION
**Pull request by dtoakley**

### CODE REVIEW

**Comment by mikelmaron** [comment]
Looks ok to me @dtoakley

Did you try running it and get the result you expected?

**Comment by mikelmaron** [comment]
Actually you may want to use the same pattern as “make_state” to handle the unlikely but possible situation where location is null

**Comment by dtoakley** [comment]
Good call -- I've refactored the make_state method to handle both cases. Let me know what you think!

File: lambda/convertActionNetworkEventCampainToGeoJSON.py [hunk 1]
```diff
@@ -90,8 +90,9 @@ def convert_event(event: Dict) -> Dict:
         'eventDate': parser.parse(event.get('start_date', '4/22/2020')).strftime('%-m/%-d/%Y'),
         'timestamp': event.get('start_date', '2020-04-22T00:00:00Z"'),
         'eventLink': event.get('browser_url', ''),
+        'localStreamLink': parse_location_data(event, 'venue'),
         'location': make_location(event),
-        'state': make_state(event),
+        'state': parse_location_data(event, 'region'),
```
Comment (meganrm) [inline lambda/convertActionNetworkEventCampainToGeoJSON.py:105]: are you going to do `city` as a separate PR?

File: lambda/convertActionNetworkEventCampainToGeoJSON.py [hunk 2]
```diff
@@ -45,15 +45,17 @@ def is_us(event: Dict) -> bool:
...
+    full_location = '{address}, {locality}, {region} {postal_code}'.format(
+        address=safestrip(location.get('address_lines')[0]),
+        locality=safestrip(location.get('locality')),
+        region=safestrip(location.get('region')),
+        postal_code=safestrip(location.get('postal_code'))
+    )
     return full_location
 
-def make_state(event: Dict) -> str:
+def parse_location_data(event: Dict, data: str) -> str:
```
Comment (meganrm) [inline lambda/convertActionNetworkEventCampainToGeoJSON.py]: instead of `data` something like `key` or `dataKey` or `keyName` would be clearer

**Review by meganrm — APPROVED** [review]

**Pull request merged**